dhall-docs/src/Dhall/Docs.hs

```diff
@@ -63,8 +65,20 @@ parseOptions =
             )
        <> Options.Applicative.value "./docs" )
     <*> parsePackageNameResolver
+    <*> parseAscii
     ) <|> parseVersion
   where
+    switch name description =
+        Options.Applicative.switch
+            (   Options.Applicative.long name
+            <>  Options.Applicative.help description
+            )
+
+    parseAscii = fmap f (switch "ascii" "Format code using only ASCII syntax")
```

Comment:
If the option affects only the code formatting, I wonder whether the option name should be more precise. Maybe `--ascii-code`?!

`--ascii` could be interpreted to apply to other parts of the documentation too.

OTOH `--ascii` is a familiar flag for many users…

Reply:
For clarity, I agree to rename it to something more specific. But I worry that people expect `--ascii-code` to also affect the markdown code sections. Anyways, I think that's the best name; something even more specific like `--ascii-rendered-code` is really long.

In addition, I don't know whether we are going to do something else with `ascii` on other places of the rendered HTML.

Reply:
It seems to me that if users want ASCII support in one place they will probably want it everywhere.  I think exposing anything more granular via the command line is probably unnecessary

Reply:
I'm not sure what other rendered texts could be ASCII encoded, but I would probably want them to be in ASCII too to match my existing code base. Unless there is a need to enable different encoding via the command line, I think `--ascii` is the best argument name as it is consistent with the `dhall` command line.

I guess the extreme case would be to document in markdown the difference between ASCII and Unicode encoding, and perhaps this special case could be addressed using magic code block marker like ```DhallAscii and DhallUnicode ?

Reply:
After thinking about this, let's keep the `--ascii` flag name, but let's update the description of the command to make it more specific:
```suggestion
    parseAscii = fmap f (switch "ascii" "Format rendered source code using only ASCII syntax")
```
At least in one place we're telling the user what is changing by using `--ascii`